Explorer/Assets/DCL/Web3/Abstract/IEthKeyOwner.cs

```diff
@@ -1,6 +1,6 @@
 using Nethereum.Signer;
 
-namespace DCL.Web3.Accounts
+namespace DCL.Web3.Abstract
 {
     public interface IEthKeyOwner
```

Comment:
I think this interface segregation is wrong. On each place you use it, you have to explicitly cast the type to get the key. I propose to delete it and add the key field into `IWeb3Account`

Reply:
Good proposal